cronjobs/values.yaml

```diff
@@ -27,18 +27,28 @@ jobs:
         chmod -R 777 /data/csv
 
         echo "Step 2: Export old data to csv files"
-        psql $DATABASE_URL -t -c "SELECT tablename FROM pg_tables WHERE tablename LIKE 'data_%';" \
-        | xargs -I {} psql $DATABASE_URL -c "\copy {} to '/data/csv/{}.csv' WITH DELIMITER ',' CSV HEADER;"
+        psql $ORAKL_DB_URL -t -c "SELECT tablename FROM pg_tables WHERE tablename LIKE 'data_%';" \
+        | xargs -I {} psql $ORAKL_DB_URL -c "\copy {} to '/data/csv/{}.csv' WITH DELIMITER ',' CSV HEADER;"
+        psql $DELEGATOR_DB_URL -t -c "SELECT tablename FROM pg_tables WHERE tablename LIKE 'transactions_%';" \
+        | xargs -I {} psql $DELEGATOR_DB_URL -c "\copy {} to '/data/csv/{}.csv' WITH DELIMITER ',' CSV HEADER;"
 
         echo "Step 3: Upload all csv files to GCS bucket."
         gcloud storage cp /data/csv/*.csv gs://orakl-db-backup-cypress
 
         echo "Step 4: Delete old data tables"
-        psql $DATABASE_URL -t -c "SELECT tablename FROM pg_tables WHERE tablename LIKE 'data_%';" \
-        | xargs -I {} psql $DATABASE_URL -c "DROP TABLE ${};"
+        psql $ORAKL_DB_URL -t -c "SELECT tablename FROM pg_tables WHERE tablename LIKE 'data_%';" \
+        | xargs -I {} psql $ORAKL_DB_URL -c "DROP TABLE {};"
+        psql $DELEGATOR_DB_URL -t -c "SELECT tablename FROM pg_tables WHERE tablename LIKE 'transactions_%';" \
+        | xargs -I {} psql $DELEGATOR_DB_URL -c "DROP TABLE {};"
```

Comment:
@jay-kim-kor Shouldn't we delete the CSV files after we uploaded them to bucket?

Reply:
Step 3 is uploading csv files part. Also this part doesn't delete csv files. It'd delete archived table after upload csv files.
The csv files will deleted with pod when job was completed or failed.

Reply:
Got it! Thank you!